src/components/select/select.template.tsx

```diff
@@ -254,14 +255,16 @@ function createMenuItems(
     return options.filter(menuOptionFilter).map((option) => {
         const selected = isSelected(option, value);
         const { text, disabled } = option;
+        const name = getIconName(option.icon);
+        const color = getIconColor(option.icon, option.iconColor);
 
         return {
             text: text,
             selected: selected,
             disabled: disabled,
             value: option,
-            icon: option.icon,
-            iconColor: option.iconColor,
+            icon: name,
+            iconColor: color,
```

Comment:
Why are we using `iconColor` here if we have deprecated it :smile: 
This is why you get the warning in the console you asked about here https://github.com/Lundalogik/lime-elements/pull/2542#discussion_r1376445815